org.civicrm.bootstrapcivicrm/scss/civicrm/contact/pages/_new-activity.scss

```diff
@@ -401,14 +439,42 @@
 
           td {
             background: $crm-white;
-            color: $gray-dark;
             border: none;
+            color: $gray-dark;
+            line-height: 18px;
```

Comment:
For this specific, it is coming from data table with line height 35px, which we reduced to 18px now for making it better.